roles/pulp/defaults/main.yml

```diff
@@ -38,6 +38,7 @@ prereq_pip_packages: []
 pulp_rhel_codeready_repo:
   - codeready-builder-for-rhel-8-x86_64-rpms
   - rhui-codeready-builder-for-rhel-8-rhui-rpms
```

Comment:
Is this old value (that I wrote) ever accurate? I might have misread it on EC2. What cloud environment did you see your value on?

Reply:
My value was taken on EC2.  I cannot judge for your old value, as I am a newcomer to AWS :)